pkg/deployer/deployer_test.go

```diff
@@ -22,13 +22,16 @@ const twinID = 11
 
 var identity, _ = substrate.NewIdentityFromEd25519Phrase(Words)
 
-func deployment1(identity substrate.Identity, TLSPassthrough bool, version uint32) gridtypes.Deployment {
+var backendURLWithTLSPassthrough = "//1.1.1.1:10"
```

Comment:
missing https?

Reply:
linked to https://github.com/threefoldtech/zos/issues/1886